stdlib/3/concurrent/futures/_base.pyi

```diff
@@ -73,7 +38,7 @@ class Future(Generic[_T]):
 
 class Executor:
     def submit(self, fn: Callable[..., _T], *args: Any, **kwargs: Any) -> Future[_T]: ...
-    def map(self, func: Callable[..., _T], *iterables: Any, timeout: float = ...) -> Iterable[_T]: ...
+    def map(self, func: Callable[..., _T], *iterables: Any, timeout: float = ..., chunksize:int = 1) -> Iterable[_T]: ...
```

Comment:
What does `...` do in this place? In the documentation, I only found it for elision of function/class bodies or arbitrary types. Does it specify something like "there is a default value, but we specify it in the child class"? Then, why should I remove it in process.pyi too? Or does it specify "there is a default value, but it doesn't belong in the typestub, it's in the actual .py file"? Then, I should probably also remove it from thread.pyi?